ui/ui-components/pages/model/[modelId]/source/[sourceId]/edit.tsx

```diff
@@ -237,6 +216,21 @@ const Page: NextPage<Props & InjectedProps> = ({
       }
     }
 
+    const { properties, examples } = await execApi<Actions.PropertiesList>(
+      "get",
+      `/properties`,
+      {
+        includeExamples: true,
+        state: "ready",
```

Comment:
I see that `unique: true` was removed when it was moved here. I don't remember a the details but I take it it's not necessary?

Reply:
Looks like a copy/paste error.  It doesn't appear to affect the bug fix to have it/not have it.  I put `unique: true` back!